tensorboard/compat/tensorflow_stub/io/gfile_test.py

```diff
@@ -82,6 +82,13 @@ def testListdir(self):
             expected_files,
             gfile.listdir(temp_dir))
 
+    def testMakeDirs(self):
+        temp_dir = tempfile.mkdtemp(prefix=self.base_temp_dir)
```

Comment:
Now that `tensorboard.test.TestCase` exists - thanks! - could we use that in this test file so it's hooked into the bazel test temp directory?

Reply:
You're right! Changing.